Search Job Records fail to parse doubles

When parsing the records from search results, the `doubleField` method fails with the following exception:

```
sumo.getRecordsForSearchJob(searchJobId, 0, 10).getRecords().get(0).doubleField("done_tasks_upper")
```

```
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:297)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NumberFormatException: For input string: "done_tasks_upper"
        at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
        at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.lang.Double.parseDouble(Double.java:538)
        at com.sumologic.client.searchjob.model.SearchJobRecord.doubleField(SearchJobRecord.java:85)
        at com.sumologic.MyCode(MyCode.java:49) 
```

--- a/src/main/java/com/sumologic/client/searchjob/model/SearchJobRecord.java
+++ b/src/main/java/com/sumologic/client/searchjob/model/SearchJobRecord.java
@@ -82,7 +82,7 @@ public class SearchJobRecord {
      * @return The value for the specified field as a double.
      */
     public double doubleField(String fieldName) {
-        return Double.parseDouble(fieldName);
+        return Double.parseDouble(stringField(fieldName));
     }
 
     /**

